habitat-lab/habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -109,6 +112,7 @@ def __init__(self, config: "DictConfig"):
         self.viz_ids: Dict[Any, Any] = defaultdict(lambda: None)
         self._handle_to_object_id: Dict[str, int] = {}
         self._markers: Dict[str, MarkerInfo] = {}
+        self._targets: Dict[str, mn.Matrix4] = {}
```

Comment:
What are those _targets? How are they relevant to KRM?

Reply:
This was a clean-up refactor, not related to KRM. IMO, `self.` variables should all be declared in __init__(), even if they are initialized in an internal function. This allows any comments and typing to be co-located. More straightforward paradigm than trying to hunt around for the first declaration of the variable in some internal function.